### PR TITLE
[WEBSITE-1215] Restructure application files and urls 

### PR DESCRIPTION
**Pull request by mattrayner**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -10,16 +10,19 @@ gem 'haml'
 # Use Puma as our web server
 gem 'puma'
 
-gem 'parliament-ruby', '~> 0.7.6'
+gem 'parliament-ruby'
+
 # Use Parliament-Opensearch to handle our Opensearch requests
-gem 'parliament-opensearch', '~> 0.2.5'
+gem 'parliament-opensearch'
```
Comment (katylouise) [inline Gemfile]: Why do we need to remove all the versions of the gems?

File: lib/parliament/search/application.rb [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      #   begin
+      #     logger.info "Making a query for '#{@query_parameter}' => '#{@escaped_query_parameter}' using the base_url: '#{request.base_url}'"
+      #     @results = request.get({ query: @escaped_query_parameter, start_page: @start_page })
+      #     @results_total = @results.totalResults
+      #
+      #     show 'search/results'
+      #   rescue Parliament::ServerError
+      #     show 'search/no_results'
+      #   end
+      # end
```
Comment (katylouise) [inline lib/parliament/search/application.rb]: Do you mean for all the above code to be commented out?

File: Gemfile (hunk 1 above)
Comment (mattrayner) [reply Gemfile]: These can probably be added back :-)

File: lib/parliament/search/application.rb (hunk 2 above)
Comment (mattrayner) [reply lib/parliament/search/application.rb]: This should be removed :-)

**Review by katylouise — APPROVED** [review]

**Pull request merged**